projects/eslint-plugin/rules/no-tui-assert-without-ng-dev-mode.js

```diff
@@ -0,0 +1,152 @@
+const traverseLogicalExpression = require('../utils/traverse-logical-expression');
+
+const MESSAGE_ID = `no-tui-assert-without-ng-dev-mode`;
+const ERROR_MESSAGE = `tuiAssert.assert must be used along with ngDevMode`;
+
+const requiredIdentifierName = 'ngDevMode';
+
+/**
+ * @type {import('eslint').Rule.RuleModule}
+ */
+module.exports = {
+    meta: {
+        type: 'problem',
+        docs: {description: ERROR_MESSAGE},
+        messages: {
+            [MESSAGE_ID]: ERROR_MESSAGE,
+        },
+        fixable: 'code',
+    },
+    create(context) {
+        return {
+            /**
+             * @param node {import('@types/estree').ExpressionStatement}
+             */
+            'CallExpression[callee.object.name="tuiAssert"][callee.property.name="assert"]'(
```

Comment:
```
[callee.object.name="tuiAssert"][callee.property.name="assert"]
```

it would be great if it could be customized

Reply:
not a problem, will implement this